pallets/tips/src/tests.rs

```diff
@@ -157,6 +159,36 @@ impl pallet_treasury::Config for Test {
     type MaxApprovals = MaxApprovals;
     type ProposalBondMaximum = ();
 }
+
+const MILLICENTS: Balance = 10_000_000_000_000;
+const CENTS: Balance = 1_000 * MILLICENTS;
+const DOLLARS: Balance = 100 * CENTS;
+
+parameter_types! {
+    pub const ClassDeposit: Balance = 100 * DOLLARS;
+    pub const InstanceDeposit: Balance = 1 * DOLLARS;
+    pub const KeyLimit: u32 = 32;
+    pub const ValueLimit: u32 = 256;
+    pub const StringLimit: u32 = 50;
+}
+
+impl pallet_uniques::Config for Test {
+    type Event = Event;
+    type ClassId = u32;
+    type InstanceId = u32;
+    type Currency = Balances;
+    type ForceOrigin = frame_system::EnsureRoot<u128>;
```

Comment:
why using a `u128`? we are using `EnsureRoot<AccountId>` elsewhere.
also `EnsureRoot` was already imported, no need to use its full path.

Reply:
i try remove "frame_system::", but EnsureRoot have no imported.

1 there are not alias like "type AccountId = u64;"
2 "type AccountId = u128; // u64 is not enough to hold bytes used to generate bounty account" in  "impl frame_system::Config for Test"